_includes/docs/extending/installing.md

```diff
@@ -0,0 +1,15 @@
+You can install extensions via the UI or the command line.
+
+### UI
```

Comment:
Hmm, the thing is, that while downloading is discouraged via the UI, Installing via UI is encouraged